### DISCUSSION TITLE
Permanently change defult number of results from 40 to 1000

### DISCUSSION THREAD

**Discussion by lamyergeier** [stashapp/stash#4717] [General]
In scenes we can choose the number of results to be shown which is by default 40. Can I change it  permanently from 40 to 1000? So that it shows 1000 results even after reboot.

**Comment by DogmaDragon** [comment]
![image](https://github.com/stashapp/stash/assets/103123951/76eafbb1-67c9-44f2-8ecc-dbc7ffb18967)

And then set the filter as default.

**Comment by Maista6969** [comment]
Yes, the number of scenes per page is a part of the default filter and you can save your own defaults
![image](https://github.com/stashapp/stash/assets/131594657/b245766f-5da2-4374-ba31-e8bb85d02199)
